src/main/scala/lc/background/taskThread.scala

```diff
@@ -5,12 +5,13 @@ import java.util.concurrent.{ScheduledThreadPoolExecutor, TimeUnit}
 import lc.core.Captcha
 import lc.core.{Parameters, Size}
 
-class BackgroundTask(captcha: Captcha, throttle: Int, timeLimit: Int) {
+class BackgroundTask(throttle: Int, timeLimit: Int) {
+
+  val captcha = new Captcha()
```

Comment:
What is the benefit of creating separate Captcha instances? Further, the `Captcha` class doesn't seem to have any state. Can it be just converted to `object` ?

Reply:
No benefit really. I just thought let the BT and Server have their own instances. 
But yeah, converting it into an object makes more sense.